lib/Runner.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 
 const AWS = require('aws-sdk');
+const EventEmitter = require('events');
 const BPromise = require('bluebird');
+const percentile = require('percentile');
 
-const INVOCATION_DELAY = 500;
 const REPORT_REGEX = /REPORT RequestId: ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\s+Duration: ([0-9.]+) ms\s+Billed Duration: ([0-9]+) ms\s+Memory Size: (\d+) MB\s+Max Memory Used: (\d+) MB\s+$/; // eslint-disable-line max-len
+const PERCENTILES = [50, 66, 75, 80, 90, 95, 98, 99];
 
-class Runner {
+class Runner extends EventEmitter {
```

Comment:
Runner class now does nothing to stdout, so can be used as lib (I want to use for serverless plugin!)